controllers/workspace_controller_sshkey_test.go

```diff
@@ -69,7 +68,8 @@ var _ = Describe("Workspace controller", Ordered, func() {
 						Key: secretKey,
 					},
 				},
-				Name: workspace,
+				Name:        workspace,
+				Description: "SSH Key",
```

Comment:
How about?
```suggestion
				Description: "SSH Key test",
```
It seems like it would make it easier to spot a zombie from a failed test.

Reply:
As discussed in Slack, we will use the `test` tag to filter out workspaces created for tests and include descriptions to better understand which tests caused a zombie workspace.